### DISCUSSION TITLE
Issue with pyreadstat installation: 'iconv.h' file not found

### DISCUSSION THREAD

**Discussion by TCcodemaster** [Roche/pyreadstat#273] [Q&A] [answered]
Hello,

I am encountering an issue while trying to install the pyreadstat package. Here are the details:

During the installation, I received the following error message:

```

C:\Users\raisi\AppData\Local\Temp\pip-install-yqj_4kpv\pyreadstat_911dec3f61ea42a79c5944d6e38fc49c\src\readstat_iconv.h(1): fatal error C1083: Cannot open include file: 'iconv.h': No such file or directory
error: command 'C:\\Program Files\\Microsoft Visual Studio\\2022\\Community\\VC\\Tools\\MSVC\\14.41.34120\\bin\\HostX86\\x64\\cl.exe' failed with exit code 2
```
I have already installed libiconv and set the appropriate environment variables, but the issue persists. This problem did not occur on my previous computer. However, I am now using a new machine, and I encountered this issue while installing packages. I am currently using Python 3.13.

Could you please provide guidance on how to resolve this issue?

Thank you for your assistance!

**Comment by ofajardo** [comment, marked as answer]
See these instructions : https://github.com/Roche/pyreadstat/blob/master/windows_compilation.md

Also here:
https://github.com/ofajardo/pyreadstat_wheels4/blob/master/appveyor.yml

Or wait a bit for the wheels for python 3.13 to be on pypi =)

**Reply by TCcodemaster** [reply]
Thank you,  I uninstalled Python version 3.13, downloaded version 3.12, and the issue was resolved

**Reply by ofajardo** [reply]
that's also a good solution. Did not mention it as I thought you needed 3.13 at any cost =). 
I just uploaded the wheels for 3.13 to pypi, so now a simple pip install should also work.

**Reply by TCcodemaster** [reply]
Thank you so much. Your contribution has made the installation process much more convenient
